### ISSUE TITLE
add an open source (un)license to this repository

### ISSUE THREAD

**Issue by streichler** [StanfordLegion/gasnet#4]
It has been observed that this repo lacks any kind of license, which technically prohibits folks from modifying or incorporating these curated-for-legion gasnet configs into derivative works.  I proposed that we use the Unlicense for this: https://choosealicense.com/licenses/unlicense/

(With the exception of the gasnet tarballs, which are redistributed under the terms of their license: https://gasnet.lbl.gov/dist-ex/license.txt )

@elliottslaughter @lightsighter @magnatelee @marcinz : as the other contributors to this repository, your explicit approval is required for this change.  Please either grant approval or raise any questions/concerns you have.  Thanks.

**Comment by bonachea** [comment]
Whatever you do, please append a copy of the latest GASNet license or otherwise loudly clarify that the GASNet tarballs are governed by their own licenses embedded in each tarball.

**Comment by elliottslaughter** [comment]
I approve.

**Comment by streichler** [comment]
> Whatever you do, please append a copy of the latest GASNet license or otherwise loudly clarify that the GASNet tarballs are governed by their own licenses embedded in each tarball.

Yes, that is the plan.  Assuming all the contributors on board, the next step will be a PR with the actual license file text, and I'll include you on the list of required approvers to be certain it is satisfactory from the gasnet team's perspective.

**Comment by lightsighter** [comment]
Approve

**Comment by magnatelee** [comment]
I approve.

**Comment by marcinz** [comment]
I approve
